parallel-install/pkg/preinstaller/preinstaller.go

```diff
@@ -25,19 +25,19 @@ package preinstaller
 import (
 	"fmt"
 	"github.com/avast/retry-go"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/config"
 	"github.com/kyma-incubator/hydroform/parallel-install/pkg/logger"
 	"io/ioutil"
 	"k8s.io/client-go/dynamic"
-	"k8s.io/client-go/tools/clientcmd"
-
 	"os"
 )
 
 // Config defines configuration values for the PreInstaller.
 type Config struct {
 	InstallationResourcePath string           //Path to the installation resources.
 	Log                      logger.Interface //Logger to be used
-	KubeconfigPath           string
+	KubeconfigPath           string           // Path to the kubeconfig file
+	KubeconfigRaw            string           // Content of the kubeconfig file
```

Comment:
Consider renaming to `KubeconfigContent` to be consistent with naming in `Config` type